charts/kyuubi/values.yaml

```diff
@@ -235,3 +235,9 @@ prometheusRule:
   enabled: false
   # Contents of Prometheus rules file
   groups: []
+
+service:
+  sessionAffinity: None
```

Comment:
what's the value candidates?

Reply:
> what's the value candidates?

ClientIP or None. Do we need to add a hint for it?

Reply:
Yes, otherwise the user has no idea how to deal with that.

Additionally, I suppose the content could be flexible, if yes, I suppose we should ref the yaml snippets directly, like `resources` in line 175

Reply:
Got it.  The content should be fixed, but I try to make a improvement. @pan3793